crates/tests-e2e/src/bitcoind.rs

```diff
@@ -75,6 +71,106 @@ impl Bitcoind {
         Ok(response)
     }
 
+    pub async fn send_multiple_utxos_to_address<F>(
```

Comment:
I had to add this bad boy to avoid weird CI bugs related to calling `sendrawtransaction` and `generatetoaddress` too many times in a row.

Reply:
Actually that was a gigantic waste of time, because an actual problem was that we cannot have more than one e2e test per file 🤦